Source/WEB_ROOT/admin/asset/assets.html

```diff
@@ -356,4 +124,17 @@ <h1>Asset Management</h1>
 <script src="../../Inventory/psQuery/psQuery.js"></script>
 <script src="../../Inventory/js/addHistory.js"></script>
 <script src="../../Inventory/js/assets.js"></script>
-<script src="../../Inventory/js/getAssetsAjax.js"></script>
\ No newline at end of file
+<script src="../../Inventory/js/getAssetsAjax.js"></script>
+<script src="../../Inventory/js/cookies.js"></script>
+
+<script>
+//console.log("Start checkCookie : " + getCookie("searchFor"));
+if(getCookie("searchFor") != "return_ALL_assets")
+{
+    //console.log("on page load, search for the cookie");
+    MakeRequest(getCookie("searchFor"));

```

Comment:
The cookie route will definitely work here John. You may want to consider asking users to enable cookies if they don't have it by default if you get bored...